lib/ssl/src/ssl.erl

```diff
@@ -3071,9 +3144,11 @@ setopts(#sslsocket{}, Options) ->
     {error, {options,{not_a_proplist, Options}}}.
 
 %%---------------------------------------------------------------
--doc(#{equiv => getstat/2}).
 -doc(#{title => <<"Utility Functions">>,
        since => <<"OTP 19.0">>}).
+-doc """
+Get statistics for the underlying TCP socket.
```

Comment:
Probably we should changes this to only the underlaying socket. As in DTLS that will be an UDP socket, and inet:getstat/2 are documented for both TCP and UDP.

Reply:
Changed.